pkgs/tools/package-management/pacman/default.nix

```diff
@@ -16,6 +16,12 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ autoreconfHook pkgconfig ];
   buildInputs = [ perl libarchive openssl zlib bzip2 lzma ];
 
+  postFixup = ''
+    substituteInPlace $out/bin/repo-add \
+      --replace bsdtar "${libarchive}/bin/bsdtar" \
+      --replace openssl "${openssl}/bin/openssl" \
```

Comment:
I have not found any openssl reference in any file.

Reply:
It used to use openssl for checksums, but switched to coreutils a while ago. We don't need to substitute coreutils programs, right?

Reply:
It does not not hurt though.